### PR TITLE
:recycle: Refactor directory structure

### PR DESCRIPTION
**Pull request by huyfififi**
@gomyway1216 and I created this repository with the bare minimum functionality, but because it's expected that we will add more features soon, we should tweak the directory structure to follow the best practices https://github.com/fastapi/full-stack-fastapi-template/tree/master
By following the design, it will be much easier to manage multiple features

### CODE REVIEW

**Comment by huyfififi** [comment]
@gomyway1216 Could you update the repository settings to allow me to review my own pull requests? I prefer not having to wait for approval

**Review by carolina-museum — APPROVED** [review]

**Pull request merged**
